2024-02-05 12:18:30.829 [info] Extension host with pid 5236 started
2024-02-05 12:18:30.829 [info] Skipping acquiring lock for /home/elisena/.config/Code/User/workspaceStorage/6442a3781aac7f0018a3b769379cf1e2.
2024-02-05 12:18:30.928 [info] ExtensionService#_doActivateExtension vscode.emmet, startup: false, activationEvent: 'onLanguage'
2024-02-05 12:18:30.934 [info] ExtensionService#_doActivateExtension ms-python.python, startup: false, activationEvent: 'onLanguage:python'
2024-02-05 12:18:31.000 [info] ExtensionService#_doActivateExtension ms-toolsai.jupyter, startup: false, activationEvent: 'onLanguage:python'
2024-02-05 12:18:31.576 [info] ExtensionService#_doActivateExtension ms-toolsai.jupyter-renderers, startup: false, activationEvent: 'api', root cause: ms-toolsai.jupyter
2024-02-05 12:18:31.583 [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*', root cause: vscode.git
2024-02-05 12:18:31.585 [info] ExtensionService#_doActivateExtension mhutchie.git-graph, startup: true, activationEvent: '*'
2024-02-05 12:18:31.701 [info] ExtensionService#_doActivateExtension vscode.git, startup: true, activationEvent: '*'
2024-02-05 12:18:31.723 [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
2024-02-05 12:18:31.781 [info] Eager extensions activated
2024-02-05 12:18:31.807 [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: false, activationEvent: 'onStartupFinished'
2024-02-05 12:18:31.809 [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
2024-02-05 12:18:31.819 [info] ExtensionService#_doActivateExtension Catppuccin.catppuccin-vsc, startup: false, activationEvent: 'onStartupFinished'
2024-02-05 12:18:31.823 [info] ExtensionService#_doActivateExtension eamodio.gitlens, startup: false, activationEvent: 'onStartupFinished'
2024-02-05 12:18:31.879 [info] ExtensionService#_doActivateExtension esbenp.prettier-vscode, startup: false, activationEvent: 'onStartupFinished'
2024-02-05 12:18:31.920 [info] ExtensionService#_doActivateExtension pdconsec.vscode-print, startup: false, activationEvent: 'onStartupFinished'
2024-02-05 12:18:31.973 [info] ExtensionService#_doActivateExtension phind.phind, startup: false, activationEvent: 'onStartupFinished'
2024-02-05 12:18:32.475 [info] ExtensionService#_doActivateExtension streetsidesoftware.code-spell-checker, startup: false, activationEvent: 'onStartupFinished'
2024-02-05 12:18:32.537 [info] ExtensionService#_doActivateExtension vscodevim.vim, startup: false, activationEvent: 'onStartupFinished'
2024-02-05 12:18:32.868 [info] ExtensionService#_doActivateExtension vscode.github-authentication, startup: false, activationEvent: 'onAuthenticationRequest:github'
2024-02-05 12:18:33.007 [error] [ms-python.python] provider FAILED
2024-02-05 12:18:33.007 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:18:33.108 [error] [ms-python.python] provider FAILED
2024-02-05 12:18:33.108 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:18:33.159 [error] [ms-python.python] provider FAILED
2024-02-05 12:18:33.160 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:18:33.163 [error] [ms-python.python] provider FAILED
2024-02-05 12:18:33.163 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:18:33.255 [error] [ms-python.python] provider FAILED
2024-02-05 12:18:33.255 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:18:33.955 [info] ExtensionService#_doActivateExtension vscode.json-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
2024-02-05 12:18:33.968 [info] ExtensionService#_doActivateExtension vscode.typescript-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
2024-02-05 12:18:34.194 [error] [ms-python.python] provider FAILED
2024-02-05 12:18:34.194 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:19:48.079 [error] [ms-python.python] provider FAILED
2024-02-05 12:19:48.079 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:15.057 [error] [ms-python.python] provider FAILED
2024-02-05 12:20:15.058 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:15.058 [error] [ms-python.python] provider FAILED
2024-02-05 12:20:15.058 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:15.584 [error] [ms-python.python] provider FAILED
2024-02-05 12:20:15.584 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:16.269 [error] [ms-python.python] provider FAILED
2024-02-05 12:20:16.269 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:22.833 [error] [ms-python.python] provider FAILED
2024-02-05 12:20:22.833 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:23.171 [error] [ms-python.python] provider FAILED
2024-02-05 12:20:23.171 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:23.989 [error] [ms-python.python] provider FAILED
2024-02-05 12:20:23.989 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:31.735 [warning] TextEditor is closed/disposed
2024-02-05 12:20:31.755 [error] [ms-python.python] provider FAILED
2024-02-05 12:20:31.755 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:31.785 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:3,$model4)
2024-02-05 12:20:31.785 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:3,$model4)
2024-02-05 12:20:31.785 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:3,$model4)
2024-02-05 12:20:31.785 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:3,$model4)
2024-02-05 12:20:31.785 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:3,$model4)
2024-02-05 12:20:33.933 [error] [ms-python.python] provider FAILED
2024-02-05 12:20:33.933 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:34.234 [error] [ms-python.python] provider FAILED
2024-02-05 12:20:34.234 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:20:37.439 [warning] TextEditor is closed/disposed
2024-02-05 12:20:37.461 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:1,$model4)
2024-02-05 12:20:37.461 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:1,$model4)
2024-02-05 12:20:37.462 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:1,$model4)
2024-02-05 12:20:37.462 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:1,$model4)
2024-02-05 12:20:37.462 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:1,$model4)
2024-02-05 12:20:37.462 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:1,$model4)
2024-02-05 12:20:37.462 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:1,$model4)
2024-02-05 12:20:37.462 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:1,$model4)
2024-02-05 12:20:37.462 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:1,$model4)
2024-02-05 12:20:37.462 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:1,$model4)
2024-02-05 12:21:22.435 [error] [ms-python.python] provider FAILED
2024-02-05 12:21:22.435 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:21:22.436 [error] [ms-python.python] provider FAILED
2024-02-05 12:21:22.436 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:21:22.935 [error] [ms-python.python] provider FAILED
2024-02-05 12:21:22.935 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:21:23.308 [error] [ms-python.python] provider FAILED
2024-02-05 12:21:23.308 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:21:23.641 [error] [ms-python.python] provider FAILED
2024-02-05 12:21:23.641 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:21:23.707 [error] [ms-python.python] provider FAILED
2024-02-05 12:21:23.707 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 12:21:42.609 [error] [ms-python.python] provider FAILED
2024-02-05 12:21:42.609 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:16:22.927 [error] [ms-python.python] provider FAILED
2024-02-05 13:16:22.927 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:17:47.442 [error] [ms-python.python] provider FAILED
2024-02-05 13:17:47.442 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:18:11.075 [error] [ms-python.python] provider FAILED
2024-02-05 13:18:11.075 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:18:11.425 [error] [ms-python.python] provider FAILED
2024-02-05 13:18:11.425 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:18:13.371 [error] [ms-python.python] provider FAILED
2024-02-05 13:18:13.371 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:18:14.174 [error] [ms-python.python] provider FAILED
2024-02-05 13:18:14.174 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:18:18.287 [error] [ms-python.python] provider FAILED
2024-02-05 13:18:18.287 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:18:20.122 [error] [ms-python.python] provider FAILED
2024-02-05 13:18:20.122 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:19:19.593 [error] [ms-python.python] provider FAILED
2024-02-05 13:19:19.594 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:16.813 [warning] TextEditor is closed/disposed
2024-02-05 13:22:16.831 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:3,$model8)
2024-02-05 13:22:16.831 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:3,$model8)
2024-02-05 13:22:16.831 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:3,$model8)
2024-02-05 13:22:16.831 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:3,$model8)
2024-02-05 13:22:16.831 [warning] Illegal argument: TextEditor(vs.editor.ICodeEditor:3,$model8)
2024-02-05 13:22:22.790 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:22.790 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:22.791 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:22.791 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:23.038 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:23.038 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:23.290 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:23.290 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:24.016 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:24.016 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:25.854 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:25.854 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:25.861 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:25.861 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:26.112 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:26.112 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:26.640 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:26.640 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:26.645 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:26.645 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:26.909 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:26.909 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:27.901 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:27.901 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:28.510 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:28.510 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:28.517 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:28.517 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:28.779 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:28.779 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:38.107 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:38.107 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:38.117 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:38.117 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:38.371 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:38.371 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:38.803 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:38.803 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:39.060 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:39.060 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:39.495 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:39.495 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:39.844 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:39.844 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:41.961 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:41.961 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:42.475 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:42.475 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:45.146 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:45.146 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:47.827 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:47.827 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:49.577 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:49.577 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:49.810 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:49.810 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 13:22:50.061 [error] [ms-python.python] provider FAILED
2024-02-05 13:22:50.061 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 14:35:08.194 [error] [ms-python.python] provider FAILED
2024-02-05 14:35:08.194 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 14:35:08.451 [error] [ms-python.python] provider FAILED
2024-02-05 14:35:08.451 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-05 14:36:21.884 [error] [ms-python.python] provider FAILED
2024-02-05 14:36:21.884 [error] Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
